app/src/main/java/edu/ucandroid/weathernotice/ui/main/MainViewModel.kt

```diff
@@ -17,10 +17,10 @@ class MainViewModel : ViewModel() {
 
     // TODO: Implement the ViewModel
     fun fetchWeatherLocations() {
-        weatherLocations = weatherService.fetchAllWeather();
+        weatherLocations = weatherService.fetchAllWeather()
```

Comment:
Lol - it's hard to get out of the habit of semi-colons :)

Reply:
Makes senses

Reply:
YES!  I agree;